test/testsimplifytokens.cpp

```diff
@@ -270,6 +271,16 @@ class TestSimplifyTokens : public TestFixture {
         ASSERT_EQUALS(expected, tokenizer.tokens()->stringifyList(nullptr, false));
     }
 
+    void combine_wstrings_Windows() {
+#if defined(_WIN32) && defined(UNICODE)
```

Comment:
That check doesn't seem to make sense. It should default to ANSI if just `isWindows()` is true. So we should test native on Windows and not Windows (with `_WIN32` check) and also test the `win32A` and `win32W` platforms.

And I just realized that we only have `win64`. I guess that is used more than the 32-bit so there is no way to actually configure the Unicode versions for 64-bit Windows scans and the two `win32` platform make even less sense and we could just deprecate those without doing any other changes whatsoever. This is such a mess...😪😶😭

Reply:
Yes it is.
So will we be dropping support for anything but 64bit + UNICODE?
It's also kind of silly that we accept two unknown macros in a row (`return _T(...) _T(...);`), but throw an error for the third.

Reply:
> So will we be dropping support for anything but 64bit + UNICODE?

At least I will try to deprecate certain things and make them more generic. But I have already been doing that in a very iterative way for quite a while now. There' just two many things which are entangled and you need to wrap your head around.

The built-in platforms are a major pain as they spill into the GUI which makes things even more complicated as they are.